patternfly/patternfly-elements
# fix: pfe-cta arrow wrap issue #679

[Resolves Issue #679](https://github.com/patternfly/patternfly-elements/issues/679)

* Link(s) to demo pages where this element can be viewed:
https://deploy-preview-765--happy-galileo-ea79c4.netlify.com//elements/pfe-cta/demo/

---

### What has changed and why
Summarize files edited as part of this MR along with a brief description of what was changed/why.

* Used nobreak space in the template + white-space:nowrap to force the arrow to stick to the last word in the CTA. Then reset the white-space on the link inside the CTA. Firefox understands the box model differently, so there is a browser specific style to limit the width of the CTA so the arrow does not get cut off.

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

Use the resize box on the pfe-cta demo page (halfway down) to test in various browsers. 
    - The arrow should always be on the same line as the last word in the cta
    - The arrow should stay next to the last word, and not float to the edge of the CTA area



### Ready-for-merge Checklist
